daemon/snapshotter/mount.go

```diff
@@ -14,7 +14,7 @@ import (
 const mountsDir = "rootfs"
 
 // List of known filesystems that can't be re-mounted or have shared layers
-var refCountedFileSystems = []string{"overlayfs", "zfs", "fuse-overlayfs"}
+var refCountedFileSystems = []string{"fuse-overlayfs", "overlayfs", "stargz", "zfs"}
```

Comment:
@ktock
Can we assume that the snapshotter name is always "stargz"?
Should we check `strings.Contains(s, "stargz")`?

Reply:
Thanks for notifying me. As of now I think it's safe to assume the snapshotter name as "stargz" as the snapshotter is normally named so in `config.toml`.